azure-mgmt-batchai/src/main/java/com/microsoft/azure/management/batchai/BatchAICluster.java

```diff
@@ -296,7 +353,10 @@
                 DefinitionStages.WithUserCredentials,
                 DefinitionStages.WithVMPriority,
                 DefinitionStages.WithSetupTask,
-                DefinitionStages.WithMountVolumes,
+                HasMountVolumes.DefinitionStages.WithMountVolumes<WithCreate>,
+                WithAppInsightsResourceId,
+                WithVirtualMachineImage,
+                DefinitionStages.WithSubnet,
```

Comment:
minor: Good to be consistent with the use of  `DefinitionStages` prefix, it is missing for `WithAppInsightsResourceId and WithVirtualMachineImage`